File: src/main/node/lambda.js
exports.handleRequest = function(event, context, callback) {
	const request = JSON.parse(event.body);
	const who = request.who;
	console.log(`Received ${who}`);
	callback(null,{statusCode: 200, body: JSON.stringify({message: `Hello ${who}!`})});
}

File: src/main/python/lambda.py
import logging, json
logger = logging.getLogger()
logger.setLevel(logging.INFO)

def handleRequest(event, context):
    logger.info('Received {}'.format(event))
    body = event['body']
    request = json.loads(body)
    message = 'Hello {}!'.format(request['who'])
    response = { 'message' : message }
    return { 'statusCode': 200 , 'body': json.dumps(response) } 
    